src/core/user-wallet.ts

```diff
@@ -137,9 +118,8 @@ export abstract class UserWallet {
     return true
   }
 
-  async updateUsername({
-    username,
-  }): Promise<{ username: string | undefined; id: string }> {
+  // TODO(remove) deprecated
+  async setUsername({ username }): Promise<{ username: string | undefined; id: string }> {
```

Comment:
Would it be possible to swap this method in the old server for the new `Accounts.setUsername` method and then delete this here?

Reply:
taking the approach of not really touching much api v1 code, and then bulk delete it shortly when we can remove the pod from the server.

otherwise there are things like type return that can have issues and I don't think it's relevant to spend time testing right now

Reply:
Ok yea makes sense 👍🏾